src/pages/admin/AdminHomepage.js

```diff
@@ -2,30 +2,10 @@ import React from 'react';
 
 function AdminHomepage() {
   return (
-    <div>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
-      <h1>Hello Admin!</h1>
+    <div className="pageContainer">
+      <div className="pageHeader">
+        <h4>Dashboard</h4>
```

Comment:
Lets say "Admin Dashboard" to be clear !!

Reply:
Done